tests/ci_scripts/run_premerge_gpu.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#
+# Copyright (c) 2020-2022, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -ex
+
+## Integration Tests
+pip install -r requirements-dev.txt
+testFolder = 'tests/integration_test'
+export PYTHONPATH=$PWD
+rm -rf /tmp/snapshot-storage
+cd ${testFolder}
```

Comment:
nit: could try `pushd` and `popd` to join and leave the current dir and wrap the following script
if you prefer `cd`, then `cd -` to return back to $OLDPWD
in this way, we could always have a consistent workspace start point for all other commands